tiledb/sm/query/readers/sparse_index_reader_base.cc

```diff
@@ -240,8 +240,14 @@ std::vector<uint64_t> SparseIndexReaderBase::tile_offset_sizes() {
           num += deletes_consolidation_no_purge_;
         }
 
+        unsigned num_vectors = schema->attribute_num() + 1 +
```

Comment:
We need to document what those vectors are...

Reply:
Done